### PR TITLE
Run the Terraform Acceptance tests on every day at midnight

### PR DESCRIPTION
**Pull request by kevinkirkup**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -16,6 +16,11 @@ pipeline {
     options {
         disableConcurrentBuilds()
     }
+
+    triggers {
+      cron('0 0 * * *')
```
Comment (Try75) [inline Jenkinsfile]: maybe 0 22 * * *? I'm just thinking not at midnight. I couldn't really find a way to see all the scheduled jobs in Jenkins to see when it was least used.

Comment (kevinkirkup) [reply Jenkinsfile]: How about '1 H * * *' - Sometime between 1am and 2am?

Comment (Try75) [reply Jenkinsfile]: "30 1 * * *"?

Comment (kevinkirkup) [reply Jenkinsfile]: Updated.

**Comment by kevinkirkup** [comment]
Also added a fix for the Locations Data Source to add the new virtual pod for Dev1.

**Pull request merged**